src/udev/udev-worker.c

```diff
@@ -248,7 +248,7 @@ void udev_broadcast_result(sd_device_monitor *monitor, sd_device *dev, EventResu
                                 (void) device_add_property(dev, "UDEV_WORKER_ERRNO_NAME", str);
                         break;
                 }
-                case EVENT_RESULT_EXIT_STATUS_BASE ... EVENT_RESULT_EXIT_STATUS_MAX:
+                case (EVENT_RESULT_EXIT_STATUS_BASE+1) ... EVENT_RESULT_EXIT_STATUS_MAX:
```

Comment:
hmm, not sure what the bit benefit of this change is, but ok.

Reply:
instead of the (int) cast up top, maybe do here: 

```c
case ((EventResult) (EVENT_RESULT_EXIT_STATUS_BASE+1)) ... EVENT_RESULT_EXIT_STATUS_MAX:
```

Reply:
EventResult enum doesn't have the value 1 so above code will fail.

Reply:
EventResult enum doesn't have the value 1 so above code will fail.

Reply:
hmm, what? what do you mean by "fail"?

Reply:
EventResult is an enum which doesn't consists any value that corresponds to 1, so if we change this it will fail in compilation.